addons/survey/static/src/js/survey_form.js

```diff
@@ -37,6 +37,9 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend({
                     self.$el.find('button[type="submit"]').click();
                 });
             }
+            if (!self.options.isStartScreen) {
+                self.$('.breadcrumb').toggleClass('d-none', false);
```

Comment:
Can you change by `addClass` or `removeClass` ? I think toggle is supposed to be used when the argument is a variable, here you know that you want to remove it.

Also, can't you do this directly in the template?